src/runtime/virtcontainers/clh.go

```diff
@@ -753,6 +753,11 @@ func (clh *cloudHypervisor) hotplugAddBlockDevice(drive *config.BlockDrive) erro
 	clhDisk.Readonly = &drive.ReadOnly
 	clhDisk.VhostUser = func(b bool) *bool { return &b }(false)
 
+	queues := int32(clh.config.NumVCPUs)
+	queueSize := int32(1024)
```

Comment:
@amshinde Can you add a comment to describe why here use 1024 as a queue size?

Reply:
Yep I agree with @liubin it'd be good to understand why you picked 1024. Having a deeper queue might bring some benefits or drawbacks depending on the use case.